### ISSUE TITLE
symlink Error: ENOTDIR: not a directory with github super-linter

### ISSUE THREAD

**Issue by ben-marengo-msmg** [kucherenko/jscpd#481]
**Describe the bug**
this issue follows https://github.com/kucherenko/jscpd/issues/192, where you mention `you should point just path to the directory with files, not single file`. unfortunately [github/super-linter](https://github.com/github/super-linter) always calls single files (because it lints the diff files only)

```
Error: ENOTDIR: not a directory, scandir '/path/to/symlink'
    at Object.readdirSync (node:fs:1380:3)
    at readdir (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.scandir/out/providers/sync.js:39:31)
    at Object.read (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.scandir/out/providers/sync.js:12:12)
    at SyncReader.scandirSync [as _scandir] (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.scandir/out/index.js:18:17)
    at SyncReader._handleDirectory (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.walk/out/readers/sync.js:28:34)
    at SyncReader._handleQueue (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.walk/out/readers/sync.js:23:18)
    at SyncReader.read (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.walk/out/readers/sync.js:15:14)
    at SyncProvider.read (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.walk/out/providers/sync.js:11:29)
    at ReaderSync.walkSync [as _walkSync] (/usr/local/lib/node_modules/jscpd/node_modules/@nodelib/fs.walk/out/index.js:20:21)
    at ReaderSync.dynamic (/usr/local/lib/node_modules/jscpd/node_modules/fast-glob/out/readers/sync.js:13:21) {
  errno: -20,
  syscall: 'scandir',
  code: 'ENOTDIR',
  path: '/path/to/symlink'
}
```

**To Reproduce**
`jscpd --noSymlinks /path/to/symlink`

or

create a repo with super-linter action
raise a pr adding a symlink

**Alternatives**
i could exclude each symlink individually from super-linter using the FILTER_REGEX_EXCLUDE env var, but this is quite a pain for many symlinks

**Expected behavior**
should not error

**Desktop (please complete the following information):**
 - OS: super-linter
 - OS Version super-linter v4
 - NodeJS Version super-linter
 - jscpd version 3.3.26

**Comment by jamiet-msm** [comment]
Hi, I'm a colleague of @ben-marengo-msmg . I've hit this problem again this week and have repro'd here: https://github.com/jamiet-msm/demo-jscpd-issue/runs/4243877994?check_suite_focus=true

**Comment by kucherenko** [comment]
fixed at jscpd@3.5.0
